src/components/Avatar/AvatarGroup.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react'
+import { Tooltip } from '../../index'
+import { VISIBLE_AVATARS_AMOUNT } from '../../consts'
+import Icon from '../Icon'
+import { Avatar } from '../index'
+import { TAvatarGroupProps } from './types'
+
+export const AvatarGroup = ({
+  avatarGroup = [],
+  onAddUser,
+  showInvisibleAvatarsAmount = true,
```

Comment:
do we need this prop? is there a case when we don't need to show invisible avatars amount?